Tests/ConsoleKitTests/CommandErrorTests.swift

```diff
@@ -2,24 +2,32 @@
 import XCTest
 
 class CommandErrorTests: XCTestCase {
-    func testMissingCommand() throws {
+    func testMissingCommand() async throws {
         let console = TestConsole()
         let group = TestGroup()
         let input = CommandInput(arguments: ["vapor"])
-        XCTAssertThrowsError(try console.run(group, input: input), "Missing command is supposed to throw") { error in
```

Comment:
A lot of these test updates are because `XCTAssertThrowsError` doesn't support async/await as far as I can tell, so we're manually try/catching.